plugins/woocommerce/src/Admin/PageController.php

```diff
@@ -124,7 +124,7 @@ public function determine_current_page() {
 			$current_url = esc_url_raw( wp_unslash( $_SERVER['REQUEST_URI'] ) );
 		}
 
-		$current_query = wp_parse_url( $current_url, PHP_URL_QUERY );
+		$current_query = wp_parse_url( $current_url, PHP_URL_QUERY ) ?: '';
```

Comment:
@SebKay Thanks for fixing this! I think it's safer to use `??` here since `0` would be considered `false`. What do you think? 🙂 

```suggestion
		$current_query = wp_parse_url( $current_url, PHP_URL_QUERY ) ?? '';
```